custom_components/localtuya/light.py

```diff
@@ -152,16 +236,51 @@ def supported_features(self):
             supports |= SUPPORT_COLOR_TEMP
         if self.has_config(CONF_COLOR):
             supports |= SUPPORT_COLOR | SUPPORT_BRIGHTNESS
+        if self.has_config(CONF_SCENE) or self.has_config(CONF_MUSIC_MODE):
+            supports |= SUPPORT_EFFECT
         return supports
 
     def __is_color_rgb_encoded(self):
         return len(self.dps_conf(CONF_COLOR)) > 12
 
+    def __is_white_mode(self):
```

Comment:
```python
@property
def is_white_mode(self):
   ...
```
Same for the others.

Reply:
Just to understand on this, as you know, i'm not a python developer 😄 
I thought the @property is to help getters/setters of internal variable. 
This method are internal helpers to find the right mode. Do we need to make this public ?

Reply:
Done